test/unit_tests/unit_test_bitmap.cpp

```diff
@@ -6,34 +6,39 @@ namespace bitmap_operation
 {
     bool WhiteGrayScaleImage()
     {
+        bool passed = true;
```

Comment:
Remove this variable and all code related to it.

Reply:
I replaced the return value with a variable so that the image would be deleted, regardless of whether or not the test passed or failed.  What do you want me to do instead?